pkg/apis/k8s/v1alpha1/nginxingresscontroller_types.go

```diff
@@ -184,9 +189,16 @@ type ReportIngressStatus struct {
 	Enable bool `json:"enable"`
 	// Specifies the name of the service with the type LoadBalancer through which the Ingress controller pods are exposed externally.
 	// The external address of the service is used when reporting the status of Ingress resources.
-	// Note: Only if ServiceType is different than LoadBalancer.
+	// Note: Only if serviceType is NodePort.
 	// +kubebuilder:validation:Optional
 	ExternalService string `json:"externalService"`
+	// Specifies the name of the IngressLink resource, which exposes the Ingress Controller pods via a BIG-IP system.
+	// The IP of the BIG-IP system is used when reporting the status of Ingress, VirtualServer and VirtualServerRoute resources.
+	// For Ingress resources only. Requires reportIngressStatus.enable set to true.
```

Comment:
Because of the following check in the code,  "Requires reportIngressStatus.enable set to true" is for all resources, not only Ingresses:
```go
if instance.Spec.ReportIngressStatus != nil && instance.Spec.ReportIngressStatus.Enable {
```

```suggestion
	// Requires reportIngressStatus.enable set to true.
```